pytentiostat/reporter.py

```diff
@@ -17,23 +21,19 @@ def save_data_to_file(data, filename="Place_Holder.csv"):
         nothing
     '''
 
-    # These will be imported from config
-    export_file_destination = "Place/holder/path"
-
     list_data = list(data)
     df = pd.DataFrame(data=list_data,
                       columns=['Time(s)', 'Voltage(V)', 'Current(mA)'])
 
     with open(filename, mode='w', newline='\n') as f:
         df.to_csv(f, index=False, header=True)
 
-    # Uncomment after config updated
-    # shutil.move(filename, export_file_destination)
+    shutil.move(filename, export_file_destination)
```

Comment:
why not just write it directly in the desired location?  Use ``full_filename = os.path.join(export_file_destination,filename)`` then ``with open(full_filename, 'w') as f:`` etc.

Reply:
- Didn't realize, you could directly write to any directly. Implemented now!